spec/features/guest/show_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe 'guest show page' do
+
+  it 'user story 1' do 
+    # As a visitor
+    # When I visit a guest's show page
+    # I see the guest's name
+    # And I see a list of all the rooms they've stayed in
+    # including the room's suite, nightly rate, and the name of the hotel that it belongs to.
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    room1 = Room.create!(hotel_id: budapest.id, suite: "Presidential", rate: 125)
+    room2 = Room.create!(hotel_id: budapest.id, suite: "Honeymoon", rate: 200)
+    charlize = Guest.create!(name:"Charlize Theron", nights: 3)
+    guest_room1 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+    guest_room2 = GuestRoom.create!(room_id: room2.id, guest_id: charlize.id)
+    visit "guests/#{charlize.id}"
+    expect(page).to have_content("Guest name: #{charlize.name}")
+    expect(page).to have_content("Rooms Stayed in:")
+    expect(page).to have_content("Suite: #{room1.suite}")
+    expect(page).to have_content("Rate: #{room1.rate}")
+    expect(page).to have_content("Hotel: #{budapest.name}")
+    expect(page).to have_content("Suite: #{room2.suite}")
+    expect(page).to have_content("Rate: #{room2.rate}")
+  end
+
+  it 'user story 2' do
+    # As a visitor
+    # When I visit a guest's show page
+    # Then I see a form to add a room to this guest.
+    # When I fill in a field with the id of an existing room
+    # And I click submit
+    # Then I am redirected back to the guest's show page
+    # And I see the room now listed under this guest's rooms.
+    # (You do not have to test for a sad path, for example if the ID submitted is not an existing room)
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    room1 = Room.create!(hotel_id: budapest.id, suite: "Presidential", rate: 125)
+    room2 = Room.create!(hotel_id: budapest.id, suite: "Honeymoon", rate: 200)
+    charlize = Guest.create!(name:"Charlize Theron", nights: 3)
+    guest_room1 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+    guest_room2 = GuestRoom.create!(room_id: room2.id, guest_id: charlize.id)
+
+    room3 = Room.create!(hotel_id: budapest.id, suite: "Economy", rate: 75)
+    visit "/guests/#{charlize.id}"
+    expect(page).to have_content("Add a Room by its ID:")
+    expect(page).to_not have_content("Suite: #{room3.suite}")
+    expect(page).to_not have_content("Rate: #{room3.rate}")
+    fill_in 'room_id', with: room3.id
+    click_on 'Submit'
+    save_and_open_page
+    expect(current_path).to eq("/guests/#{charlize.id}")
+    expect(page).to have_content("Suite: #{room3.suite}")
+    expect(page).to have_content("Rate: #{room3.rate}")
+  end
+end
```

Comment:
I know that during an IC, time is of the essence but I would like to see some better test set up. `describe 'guest show page'` is okay but I would like to see some additional `describe` blocks being used to set up your tests.

`it 'user story 1'` doesn't tell me anything about the test. Even though you have the user stories commented in those `it` blocks, if I were to run the tests with documentation output, the output at first glance wouldn't tell me much more than there is a `user story 1` and a `user story 2`.

Beyond that, the tests are set up perfectly, and I like how you used string interpolation in your assertions to give them a little more weight and less ambiguity.

Only recommendation other recommendation I would make is to start working in `within` blocks into your tests.